### DISCUSSION TITLE
snapshot vs snapshots

### DISCUSSION THREAD

**Discussion by csicky** [capawesome-team/capacitor-firebase#689] [General]
There is this code in docs:

```
const addCollectionSnapshotListener = async () => {
  const callbackId = await FirebaseFirestore.addCollectionSnapshotListener(
    {
      reference: 'users',
    },
    (event, error) => {
      if (error) {
        console.error(error);
      } else {
        console.log(event);
      }
    }
  );
  return callbackId;
};
```

After some research online (because I could not find what is the event in the documentation), I found that the callback has on it a field named `snapshots` and the other named `addDocumentSnapshotListener` has snapshot.

This is a bit unexpected since the event is one snapshot regardless of being called on one document or on a collection. 
On official web implementation it is onSnapshot regardless of being used on a collection or document so I wonder whys would it be snapsot**s** here? One snapshot has doc changes.

**Comment by csicky** [comment]
https://github.com/capawesome-team/capacitor-firebase/blob/main/packages/firestore/src/web.ts#L252
It just feels strange to call this snapshot**s** when a snapshot is a point in time, not a collection of documents.

**Reply by robingenz** [reply]
Yes, `documents` would probably have been a more appropriate name.
